src/main/kotlin/no/nav/familie/ef/sak/mapper/IverksettingDtoMapper.kt

```diff
@@ -62,15 +62,13 @@ class IverksettingDtoMapper(private val arbeidsfordelingService: Arbeidsfordelin
 
         val fagsak = fagsakService.hentFaksakForBehandling(behandling.id)
         val vedtak = vedtakService.hentVedtak(behandling.id)
-        val forrigeBehandlingId = behandlingRepository.finnSisteIverksatteBehandling(behandling.fagsakId)
         val saksbehandler =
                 behandlinghistorikkService.finnSisteBehandlingshistorikk(behandling.id, StegType.SEND_TIL_BESLUTTER)?.opprettetAv
                 ?: error("Kan ikke finne saksbehandler på behandlingen")
         val tilkjentYtelse = tilkjentYtelseService.hentForBehandling(behandling.id)
         val vilkårsvurderinger = vilkårsvurderingRepository.findByBehandlingId(behandling.id)
 
-
-        val behandlingsdetaljer = mapBehandlingsdetaljer(behandling, vilkårsvurderinger, forrigeBehandlingId)
+        val behandlingsdetaljer = mapBehandlingsdetaljer(behandling, vilkårsvurderinger, behandling.forrigeBehandlingId)
```

Comment:
Sender inn behandling - trenger vi sende inn behandling.forrigeBehandlingId?

Reply:
Ja, vi sender den både til DVH og brukes for å hente ut forrige behandlingen sine andeler, sånn att iverksetting mot oppdrag blir riktig :)

Reply:
Tror Mattis mener at den har du allerede - du sender inn hele `behandling`-objektet og kan plukke ut forrigeBehandlingId derfra :)

Reply:
Aaah 👍

Reply:
Fikset i https://github.com/navikt/familie-ef-sak/pull/826/commits/e20d2eb67321ee8a9e72996ca82fc9375c19231f